tools/travis-ci/bazel.rc

```diff
@@ -2,13 +2,13 @@
 startup --host_jvm_args=-Xmx2500m
 startup --host_jvm_args=-Xms2500m
 startup --batch
-test --ram_utilization_factor=10
+test --ram_utilization_factor=10 --test_output=errors
```

Comment:
our convention is put each option in a separate line to improve readability. please put the test option in a separate line.